cmd/intervalOverlap/intervalOverlap.go

```diff
@@ -101,7 +102,6 @@ func main() {
 	if len(flag.Args()) != expectedNumArgs {
 		flag.Usage()
 		log.Fatalf("Error: expecting %d arguments, but got %d\n\n", expectedNumArgs, len(flag.Args()))
-		return
```

Comment:
Just to confirm, this return doesn't need to be here? We're in a main function

Reply:
Correct, the fatal will kill it before the return.